src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.tsx

```diff
@@ -200,6 +204,16 @@ const ControlsSection = (props: {
         </div>
       </div>
     );
+  } else if (haveAllJobsFailed(submission)) {
+    return (
+      <div className={styles.controlsSection}>
+        <DeleteButton onClick={onDelete} disabled={isInDeleteMode} />
+        <div className={styles.errorMessage}>
+          <span>{ALL_JOBS_FAILED_WARNING}</span>
+          <QuestionButton helpText={<FailedSubmissionHelpText />} />
```

Comment:
Didn't we say that we are removing the question button? I can't remember.

![image](https://user-images.githubusercontent.com/6834224/227477796-a87fd5f2-d66c-4efa-a35f-5b48025eba32.png)